lisa/notifiers/junit.py

```diff
@@ -302,18 +310,27 @@ def _add_test_case_result(
 
         if message.status == TestStatus.FAILED:
             failure = ET.SubElement(testcase, "failure")
-            failure.attrib["message"] = message.message
-            failure.text = message.stacktrace
-
+            if runbook.include_failed_messages:
+                failure.attrib["message"] = message.message
+                # failure.text = message.stacktrace
+                if str(message.stacktrace) != "":
```

Comment:
`if message.stacktrace:` is enough for None, "".

Reply:
updated